tests/ftest.sh

```diff
@@ -23,7 +23,7 @@ export PERF8_TRACE=${PERF8_TRACE:-False}
 export REFRESH_RATE="${REFRESH_RATE:-5}"
 export DATA_SIZE="${DATA_SIZE:-medium}"
 export RUNNING_FTEST=True
-export VERSION='8.10.0-SNAPSHOT'
+export VERSION='8.11.0-SNAPSHOT'
```

Comment:
```suggestion
export VERSION='8.11.1-SNAPSHOT'
```
?

Reply:
Theoretically yeah, in practice we don't care much as we provide compatibility on major/minor version? 🤷